src/helpers/shared/user-login.js

```diff
@@ -26,10 +26,7 @@ const axiosInstance = axios.create({
 
 const resolveInterceptor = (response) => response.data || response;
 const errorInterceptor = (error = {}) => {
-  const requestId = error.response?.headers['x-rh-insights-request-id'];
```

Comment:
this was added to the notifications frontend-component, but it had the incorrect parameter for the request id. 
We should leave this in place until my PR for the frontend components is merged and released.

Reply:
Right, I've removed it because of tests debugging I forgot to restore the line. :+1: